Purchases/Misc/Obsoletions.swift

```diff
@@ -236,3 +236,9 @@ public extension Purchases {
 @available(watchOS, obsoleted: 1, renamed: "CustomerInfo")
 @available(macOS, obsoleted: 1, renamed: "CustomerInfo")
 @objc(RCPurchaserInfo) public class PurchaserInfo: NSObject { }
+
+@available(iOS, obsoleted: 1, renamed: "StoreTransaction")
+@available(tvOS, obsoleted: 1, renamed: "StoreTransaction")
+@available(watchOS, obsoleted: 1, renamed: "StoreTransaction")
+@available(macOS, obsoleted: 1, renamed: "StoreTransaction")
+@objc(RCTransaction) public class Transaction: NSObject { }
```

Comment:
oh nice! I was actually not sure we'd be able to unify with this one too, I was worried we'd end up with a confusingly-similar couple of entities

Reply:
Yeah :) the only challenge was that this required all properties to be non-`Optional`, but I think that made everything better.